src/pages/Mypage/Order/Order.js

```diff
@@ -1,7 +1,271 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import ReactDaumPost from 'react-daumpost-hook';
+import { API } from '../../../config';
+import {
+  Card,
+  CardMedia,
+  TextField,
+  Button,
+  Input,
+  InputLabel,
+  FormControl,
+} from '@mui/material';
+import PropTypes from 'prop-types';
+import { IMaskInput } from 'react-imask';
+import * as S from './Order.style';
+
+//MUI INPUT 전화번호 자동 하이픈 입력 함수
+const TextMaskCustom = React.forwardRef(function TextMaskCustom(props, ref) {
+  const { onChange, ...other } = props;
+  return (
+    <IMaskInput
+      {...other}
+      mask="000-0000-0000"
+      definitions={{
+        '#': /[1-9]/,
+      }}
+      inputRef={ref}
+      onAccept={value => onChange({ target: { name: props.name, value } })}
+      overwrite
+    />
+  );
+});
+TextMaskCustom.propTypes = {
+  name: PropTypes.string.isRequired,
+  onChange: PropTypes.func.isRequired,
+};
 
 const Order = () => {
-  return <div>Order</div>;
+  const [orderData, setOrderData] = useState({});
+  const [loading, setLoding] = useState(true);
+  const [addressData, setAddressData] = useState({
+    bidId: Number.MAX_SAFE_INTEGER,
+    phoneNumber: '',
+    street: '',
+    address: '',
+    zipcode: '',
+    price: Number.MAX_SAFE_INTEGER,
+  });
+  const { street, address, zipcode, phoneNumber } = addressData;
+  const {
+    imageUrl,
+    userName,
+    userEmail,
+    itemName,
+    authorName,
+    bidPrice,
+    shippingFee,
+    totalPrice,
+  } = orderData;
+  const orderPriceList = [
+    {
+      id: 1,
+      name: '낙찰금액',
+      낙찰금액: bidPrice,
+    },
+    {
+      id: 2,
+      name: '배송비',
+      배송비: shippingFee,
+    },
+    {
+      id: 3,
+      name: '부가세',
+      부가세: '0원',
+    },
+  ];
+
+  const handleAddressDatailChange = e => {
+    const { name, value } = e.target;
+    setAddressData({ ...addressData, [name]: value });
+  };
+
+  const postConfig = {
+    onComplete: data => {
+      setAddressData(prev => ({ ...prev, address: data.address }));
+      setAddressData(prev => ({ ...prev, zipcode: data.zonecode }));
```

Comment:
```suggestion
      setAddressData(prev => ({ ...prev, address: data.address, zipcode: data.zonecode }));
```
이렇게 한번에 해주는건 안되는건가요?

Reply:
확인해보니 정상적으로 됩니다!